js/private/js_run_devserver.mjs

```diff
@@ -88,6 +89,14 @@ async function syncRecursive(src, dst) {
                 mkdirpSync(path.dirname(dst))
             }
             await fs.promises.copyFile(src, dst)
+            if (writePerm) {
+                // add write permissions to file (preserving executable permissions if file has them)
+                if (await fs.promises.access(dst, fs.constants.X_OK)) {
```

Comment:
Can we get the permissions and add read instead of hardcoding the `755` and `644`?

Something such as (I haven't tested this 😅)...:
```
const p = await fs.promises.access(dst);
await fs.promises.chmod(dst, p | fs.constants.S_IXUSR)
```

Reply:
Its a temporary sandbox of files so I think the hard-coded values would be fine but I'll try your suggestion out. Thanks